lib/galaxy/webapps/galaxy/buildapp.py

```diff
@@ -128,6 +129,9 @@ def paste_app_factory( global_conf, **kwargs ):
 
     register_postfork_function(postfork_setup)
 
+    for th in threading.enumerate():
+        if th.is_alive():
+            log.debug("Prior to webapp return, Galaxy thread %s is alive.", th)
```

Comment:
Somehow my brain translated "prior to webapp return" to "prior to fork". Disregard my comment...